### PR TITLE
Add dependency on console distribution

### PR DESCRIPTION
**Pull request by adammitchelldev**
## What is the goal of this PR?

Part of the effort to use distributable artefacts instead of building executable dependencies through source from other repos (where the dependencies might leak).

## What are the changes implemented in this PR?

- Depend on distribution version of console

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -438,6 +450,18 @@ jobs:
           bazel run @graknlabs_build_tools//ci:release-validate-deps -- \
             graknlabs_common graknlabs_graql graknlabs_protocol graknlabs_console
```
Comment (haikalpribadi) [inline .circleci/config.yml:451]: This should be updated to take in `@graknlabs_console_distribution` now, and make sure it still works.

File: dependencies/graknlabs/BUILD [hunk 2]
```diff
@@ -13,4 +13,6 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
\ No newline at end of file
+#
+
+exports_files(["graknlabs_console.BUILD"], visibility = ["//visibility:public"])
```
Comment (haikalpribadi) [inline dependencies/graknlabs/BUILD:18]: new line

File: .gitignore [hunk 3]
```diff
@@ -59,4 +59,8 @@ grakn-test/test-integration/benchmarks/
 
 
 # Python cache #
-__pycache__
\ No newline at end of file
+__pycache__
+
+
+# bazel-local #
+.local_deps*
```
Comment (haikalpribadi) [inline .gitignore:66]: new line

File: dependencies/graknlabs/BUILD (hunk 2 above)
Comment (haikalpribadi) [inline dependencies/graknlabs/BUILD:18]: Why do we need this exported file, @adammitchelldev ? It doesn't show up in this PR either.

Comment (adammitchelldev) [reply dependencies/graknlabs/BUILD:18]: Ah, was for a previous method, good catch

**Pull request merged**